data.json

```diff
@@ -1324,5 +1324,11 @@
     "githubUsername": "Krishnapro",
     "imageUrl": "https://user-images.githubusercontent.com/59277887/132136661-d62a6664-0fa3-437d-b97d-ea41ca21c61f.png",
     "issueNumber": 702
+  },
+  {
+  "name": "Omkar Kulkarni",
+  "githubUsername": "Omkar0114",
+  "imageUrl": "https://www.dropbox.com/s/6myquj7ck6gflr2/Screenshot%202021-09-09%2012.07.50.png?dl=0",
```

Comment:
please use a github image link here instead of a dropbox link. you can create it by simply pasting a screenshot of your profile in your issue